stats-service/server/src/main/java/ru/practicum/server/model/EndpointHit.java

```diff
@@ -0,0 +1,34 @@
+package ru.practicum.server.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "endpoint_hits")
+@Data
```

Comment:
В аннотации Data есть реализация методов equals & hashCode и это очень важные методы для сущности БД, поэтому не стоит отдавать их реализацию на откуп сторонней библиотеки ломбок, а лучше сделать их самостоятельно. В целом  достаточно вместо data ставить Getter & Setter

Подробнее здесь:
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/

ключевое ниже

Your equals() method should return false if the primary key value of both entity objects is null. But Lombok’s equals() method returns true. Because of that, you can’t add two new entity objects to a Set

Метод equals должен возвращать false, когда первичные ключи равны null, но ломбок дает true. По этой причине не будет возможности добавить в Set 2 новых entity объекта

```java
@Override
public void run(String... args) throws Exception {
   Person person1 = new Person();
   Person person2 = new Person();

   System.out.println(person1.equals(person2)); // true
}

@Data
class Person {
    private String name;
}
```

Reply:
Для диплома требование обязательное